docs/bugs_and_glitches.md

```diff
@@ -720,6 +728,45 @@ This bug existed for all battles in Gold and Silver, and was only fixed for sing
 ```
 
 
+## NPC use of Full Heal does not cure confusion status
+
+([Video]())
+
+**Fix:** Edit `AI_HealStatus` and `EnemyUsedFullRestore` in [engine/battle/ai/items.asm](https://github.com/pret/pokecrystal/blob/master/engine/battle/ai/items.asm):
```

Comment:
Saying "`EnemyUsedFullRestore` and `AI_HealStatus`" would match their order in the file. Also, to keep the comments consistent, either remove the whole "Bug:" comment or change it to address both:

```
	; Bug: this should reset SUBSTATUS_NIGHTMARE and SUBSTATUS_CONFUSED too
	; Uncomment the lines below to fix
	; ld hl, wEnemySubStatus1
	; res SUBSTATUS_NIGHTMARE, [hl]
	; ld hl, wEnemySubStatus3
	; res SUBSTATUS_CONFUSED, [hl]
```

Reply:
Will do the latter to make it easier to apply the fix. Duly noted.

Reply:
Applied in the latest commit.